lib/deploy.js

```diff
@@ -74,22 +77,40 @@ Please check configuration before continuing
 }
 
 function deploy (
-  logger /* : any */,
-  bundleUrl /* : string */,
+  bundleKey /* : string */,
+  accessToken /* : string */,
   stage /* : string */,
   serviceSettings /* : BMServerSettings */
 ) /* : Promise<void> */ {
-  // TODO Call Server CLI Service to deploy
-  logger.log(logSymbols.success, `
-Deployed:
-  Bundle Url: ${bundleUrl}
-
-To:
-  Deployment Url: ${serviceSettings.deploymentUrl}
-  Stage: ${stage}
-  Target Origin: https://example.com
-`)
-  return Promise.resolve()
+  const stopUpdates = logUpdates(() => `Deploying project - this may take several minutes...`)
+  return new Promise((resolve, reject) => {
+    request({
+      auth: {
+        bearer: accessToken
+      },
+      json: {
+        bundleBucket: serviceSettings.bucket,
+        bundleKey,
+        targetOrigin: 'https://example.com' // TEMP, needed for authorisation
```

Comment:
I hadn't picked up on this before, but most of the tooling I've seen uses `// TODO` or a few other keywords for this sort of reminder: http://eslint.org/docs/developer-guide/code-conventions#comment-annotations

`// TEMP` is not incorrect or anything, so don't bother changing this now. Just something to think about and discuss as a group. Maybe TEMP is better? /shrug